test/test_mnist.py

```diff
@@ -22,9 +22,9 @@ def fetch_mnist():
 
 # create a model
 class TinyBobNet:
```

Comment:
Instead of adding GPU here, can you use the helper to convert it after you create it?

Reply:
I am not sure what you want here.

If by helper function you mean `.cpu` and `.gpu`,  it seems non-trivial to apply the functions for each of the `Tensor` parameters inside the models.

If you just don't want the GPU kwarg, perhaps I can create a base class that has an auto gpu-cpu conversion for the model?

Perhaps I am missing something obvious?

Reply:
```
[x.cuda_() for x in get_parameters(model)]
```

Reply:
This should be resolved now.

The optimizers also create Tensor objects on initialization. This means that all the optimizer parameters needed to be converted. To account for this, I updated the get_parameters function.